main.go

```diff
@@ -178,6 +179,26 @@ func main() {
 		gateway: gateway,
 	}
 
+	// Configure metrics
+	metricsHost := os.Getenv(statsdHostVariable)
+	metricsPort := os.Getenv(statsdPortVariable)
+	metricsTimeout, err := strconv.ParseInt(os.Getenv(statsdTimeoutVariable), 10, 64)
+	if err != nil {
+		log.Fatalf("Failed parsing metrics timeout: %s", err)
+		metricsTimeout = 100
```

Comment:
This line could not be reached, because `Fatalf` will call `os.Exit(1)`

Reply:
Good catch! Fixed.